core/options.js

```diff
@@ -54,6 +54,14 @@ Blockly.Options = function(options) {
     if (hasTrashcan === undefined) {
       hasTrashcan = hasCategories;
     }
+    var maxTrashcanContents = options['maxTrashcanContents'];
+    if (hasTrashcan) {
+      if (maxTrashcanContents === undefined) {
+        maxTrashcanContents = Infinity;
```

Comment:
Let's make the default 32 so there isn't just an infinite growing list of blocks in the trashcan.